src/constants.ts

```diff
@@ -1,6 +1,3 @@
-import * as coreEvents from 'storybook/internal/core-events';
-import { experimental_getStatusStore } from 'storybook/manager-api';
```

Comment:
This was really bad, I don't know why I thought this was a good idea initially. Because the preset also imports from this `constants` file, it could result in the preset pulling in `manager-api` too.